File: app/src/entourage/java/social/entourage/android/api/model/tape/EntouragePoiRequest.java
package social.entourage.android.api.model.tape;

import social.entourage.android.api.model.guide.Poi;
import social.entourage.android.api.tape.Events;

/**
 * Events specific to Entourage app
 * Created by Mihai Ionescu on 16/04/2018.
 */
public class EntouragePoiRequest extends Events {

    /**
     * Event signaling that poi view is requested
     */

    public static class OnPoiViewRequestedEvent {

        private Poi poi;

        public OnPoiViewRequestedEvent(Poi poi) {
            this.poi = poi;
        }

        public Poi getPoi() {
            return poi;
        }
    }

}

File: app/src/main/java/social/entourage/android/api/model/tape/EntouragePoiRequest.kt
package social.entourage.android.api.model.tape

import social.entourage.android.api.model.guide.Poi
import social.entourage.android.api.tape.Events

/**
 * Events specific to Entourage app
 * Created by Mihai Ionescu on 16/04/2018.
 */
class EntouragePoiRequest : Events() {
    /**
     * Event signaling that poi view is requested
     */
    class OnPoiViewRequestedEvent(val poi: Poi)
}